components/filter/filter.js

```diff
@@ -49,11 +49,11 @@ function addSpaceListener() {
 	if (spaceListenerAdded) return;
 	spaceListenerAdded = true;
 	document.addEventListener('keydown', e => {
-		if (e.keyCode !== 32) return;
+		if (e.key !== ' ') return;
```

Comment:
My IDE was giving me warnings since `keyCode` is deprecated. This is functionally equivalent.